tools/bug-report/pkg/processlog/processlog.go

```diff
@@ -61,7 +65,15 @@ func getTimeRange(logStr string, start, end time.Time) string {
 	var sb strings.Builder
 	write := false
 	for _, l := range strings.Split(logStr, "\n") {
-		t, _, _, valid := processLogLine(l)
+		var t *time.Time
+		var valid bool
+
+		if isJSONLog(logStr) {
```

Comment:
nit: make a parseLog function that does this for us so we don't need this on both calls